nav2_controller/src/controller_server.cpp

```diff
@@ -228,6 +232,7 @@ ControllerServer::on_configure(const rclcpp_lifecycle::State & state)
 
   double costmap_update_timeout_dbl;
   get_parameter("costmap_update_timeout", costmap_update_timeout_dbl);
+  tracking_feedback_pub_ = create_publisher<nav2_msgs::msg::TrackingFeedback>("tracking_feedback");
```

Comment:
Ditto here, put this below `vel_publisher_`. This is in the middle of the costmap update timeout param setting, I'm not sure why you'd put this here...